moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.h

```diff
@@ -92,7 +92,7 @@ class PR2ArmIKSolver : public KDL::ChainIkSolverPos
 #if KDL_VERSION_LESS(1, 4, 0)
   void updateInternalDataStructures();
 #else
-  void updateInternalDataStructures() override;
+  void updateInternalDataStructures();
```

Comment:
Why should override be removed? This condition shouldn't be broken by the port and we should try to avoid loosening these kind of checks. Also, the if/else block is redundant with this change.

Reply:
updateInternalDataStructures depends from orocos_kdl, we can see it on melodic here http://docs.ros.org/melodic/api/orocos_kdl/html/classKDL_1_1ChainJntToJacSolver.html#a8c6fe8ec5cd73c65699143e4a5df9bef, the problem is, that this function is not implemented on ros2, looking into https://github.com/ros2/orocos_kinematics_dynamics/tree/ros2/orocos_kdl it seems like is no more needed. This function is used only on the tests, and it seems like the function is empty.

I removed the entire function and the define here: https://github.com/ros-planning/moveit2/pull/60/commits/02f0eb363369b8954d154a4cd91a3a9eb786d119